src/server/helpers/mailer.js

```diff
@@ -0,0 +1,14 @@
+import config from '../../common/helpers/config.js';
+import nodemailer from 'nodemailer';
+
+
+const {mailConfig} = config;
+
+function sendEmail({from, html, subject, to}) {
```

Comment:
Let's do a few things to improve this function:
1. Change the file extension to `.ts` so that we can add typescript types
2. Take each of the inputs as a separate argument, rather than requiring to know the expected structure of the object
3. Add types for each argument

We'll have something like this: `function sendEmail(from:string, html:string|JSX.Element, subject:string, to:string) {`

And finally 4. Adding JSDoc style documentation like you'll see [here](https://github.com/bookbrainz/bookbrainz-site/blob/master/src/api/helpers/entityLoader.ts#L23-L44) for example

That way when we import the function in another typescript file we'll get full completion, type checking and documentation !

Reply:
sorry for the delay , actually i am in midle of my semester exams ;)

Reply:
i will be making the changes soon